core/src/main/java/org/apache/carbondata/core/datastore/chunk/store/impl/safe/AbstractNonDictionaryVectorFiller.java

```diff
@@ -168,6 +170,31 @@ public LongStringVectorFiller(int numberOfRows, int actualDataLength) {
   }
 }
 
+class BinaryVectorFiller extends AbstractNonDictionaryVectorFiller {
+
+  public BinaryVectorFiller(int numberOfRows) {
+    super(numberOfRows);
+  }
+
+  @Override public void fillVector(byte[] data, CarbonColumnVector vector) {
```

Comment:
This code is the same as LongStringVectorFiller , please use the same class, no need to add new class with same duplicated code.

Reply:
on, removed